spec/src/main/asciidoc/configaccessor.asciidoc

```diff
@@ -32,17 +32,15 @@ This can also be handled via `ConfigAccessor` as shown in the following example:
 .Simple example of ConfigAccessor
 [source,java]
 -----------------------------------------------------------------
-String userName = config.access("user.name").getValue();
+String userName = config.access("user.name", String.class).build().getValue();
 -----------------------------------------------------------------
 
 Additional to this the `ConfigAccessor` also allows for far more complex control over the access to configured values.
 
-The call to `config.access(..)` returns a `ConfigAccessor`.
+The call to `config.access(..)` returns a `ConfigAccessor.Builder`.
```

Comment:
ConfigAccessorBuilder

Reply:
No, ConfigAccess.Builder is correct. It's an inner interface.